Jellyfin.Server/StartupOptions.cs

```diff
@@ -1,43 +1,47 @@
-namespace Emby.Server.Implementations
-{
-    using CommandLine;
+using CommandLine;
+using Emby.Server.Implementations.ParsedStartupOptions;
 
+namespace Jellyfin.Server
+{
     /// <summary>
     /// Class used by CommandLine package when parsing the command line arguments.
     /// </summary>
-    public class StartupOptions
+    public class StartupOptions : IStartupOptions
     {
         [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (databases files etc.).")]
-        public string PathData { get; set; }
+        public string DataDir { get; set; }
 
         [Option('c', "configdir", Required = false, HelpText = "Path to use for config data (user policies and puctures).")]
-        public string PathConfig { get; set; }
+        public string ConfigDir { get; set; }
 
         [Option('l', "logdir", Required = false, HelpText = "Path to use for writing log files.")]
-        public string PathLog { get; set; }
-
+        public string LogDir { get; set; }
 
         [Option("ffmpeg", Required = false, HelpText = "Path to external FFmpeg exe to use in place of built-in.")]
-        public string FFmpeg { get; set; }
+        public string FFmpegPath { get; set; }
 
         [Option("ffprobe", Required = false, HelpText = "ffmpeg and ffprobe switches must be supplied together.")]
-        public string FFprobe { get; set; }
-
+        public string FFprobePath { get; set; }
 
         [Option("service", Required = false, HelpText = "Run as headless service.")]
-        public bool Service { get; set; }
+        public bool IsService { get; set; }
 
         [Option("noautorunwebapp", Required = false, HelpText = "Run headless if startup wizard is complete.")]
-        public bool NoAutoRunWebApp { get; set; }
+        public bool AutoRunWebApp { get => !NoautoRunWebApp; set => NoautoRunWebApp = value; }
```

Comment:
This is a dirty hack.

Reply:
I played with alternatives but VS preferred this one.  What do you suggest?

Reply:
On reflection, we decided to revert this change and stay with NoAutoRunWebApp.